src/classes/window/fullscreen.vala

```diff
@@ -98,6 +98,7 @@ namespace pdfpc.Window {
                         this.screen_geometry.width /= 2;
                         this.screen_geometry.height /= 2;
                 }
+                this.resizable = false;
```

Comment:
This is the only line I'm worried about because window managers are so picky about this stuff.  This works for me though.